libcxx/include/__configuration/compiler.h

```diff
@@ -33,8 +33,8 @@
 // Warn if a compiler version is used that is not supported anymore
 // LLVM RELEASE Update the minimum compiler versions
 #  if defined(_LIBCPP_CLANG_VER)
-#    if _LIBCPP_CLANG_VER < 1700
-#      warning "Libc++ only supports Clang 17 and later"
+#    if _LIBCPP_CLANG_VER < 1800
+#      warning "Libc++ only supports Clang 18 and later"
```

Comment:
The buildkite builds fail because AIX, arm, FreeBSD use Clang 17 still:
https://buildkite.com/llvm-project/libcxx-ci/builds/37764#0192014a-8840-47ea-b2bf-31aa8dd679de/2158-2953

Reply:
Great hint! Thanks. I did not yet finalized this PR (hence the draft) as we're still working on updating the CI images. This is not yet finished. As long as the images for the CI builders are not updated, this PR will not work. I keep you updated.